app/src/androidTest/java/net/skyscanner/backpack/compose/barchart/BpkBarchartTests.kt

```diff
@@ -131,14 +139,15 @@ class BpkBarchartTests : BpkSnapshotTest() {
 
   @Test
   fun selected() {
-    composed {
-      val model = BpkBarChartModel(caption = "Bar chart", items = createMonth(Month.JANUARY))
+    val model = BpkBarChartModel(caption = "Bar chart", items = createMonth(Month.JANUARY))
+    composeTestRule.setScreenshotContent {
       BpkBarChart(
         model = model,
         selected = model.items[10],
         onSelectionChange = {},
       )
     }
+    composeTestRule.onNodeWithText(model.items[10].values?.text!!).assertIsDisplayed()
```

Comment:
how come it's only this test that needs to wait for the content?

Reply:
We're waiting for the initial composition to get anchor position plus there's animation involved